plugins/permission-react/src/apis/IdentityPermissionApi.ts

```diff
@@ -35,13 +35,13 @@ export class IdentityPermissionApi implements PermissionApi {
   ) {}
 
   static create(options: {
-    configApi: Config;
-    discoveryApi: DiscoveryApi;
-    identityApi: IdentityApi;
+    config: Config;
+    discovery: DiscoveryApi;
```

Comment:
@rugvip @freben I think the `Api` suffixes were removed here for consistency with PermissionClient, but since this is front-end code, should we retain them here?

Reply:
Yeah good question, I was actually thinking about the same thing today - it kinda feels nicer to not have the api suffix, especially since the `config` one just feels wonky to have as `configApi`. Then again, you're very likely to do `const fooApi = useApi(fooApiRef)`, which makes for a good `create(config, { fooApi })` instead of `create(config, { foo: fooApi })`... but that's a pretty minor convenience.

@backstage/maintainers bikeshed a bit? :D